src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala

```diff
@@ -148,12 +152,23 @@ class RiskAnalysisValidationSpec extends AnyWordSpecLike {
         )
       )
 
-      val result: ValidationResult[RiskAnalysisFormSeed] = RiskAnalysisValidation.validate(riskAnalysis, true)
+      val result: ValidationResult[RiskAnalysisFormSeed] =
+        RiskAnalysisValidation.validate(riskAnalysis, true)(TenantKind.PA)
 
       verifyValidationFormResult(result, expected)
 
     }
 
+    "fail if version does not exists" in {
+      val riskAnalysis = SpecData.validRiskAnalysis2_0
```

Comment:
I suggest this minor change, otherwise when the v.2.0 of the private risk analysis will be released, this test will fail
```suggestion
      val riskAnalysis = SpecData.validRiskAnalysis2_0.copy(version = "9999.0")
```

Reply:
However test has success because PRIVATE Tenant has only version 1.0 and therefore version 2.0 does not exists.

Reply:
yes, for now